splink/settings.py

```diff
@@ -480,6 +487,18 @@ def _not_trained_messages(self):
         messages = []
         for c in self.comparisons:
             messages.extend(c._not_trained_messages)
+        if self._lambda_is_default:
+            messages.extend(
+                [
+                    """The probability_two_random_records_match has been set to the
```

Comment:
I think it probably makes sense to clarify that this is from the settings object. This could be as simple as:

```suggestion
                    """The 'probability_two_random_records_match' setting has been set to the
```

Reply:
I think the way the message is currently formatted with """ will also mean that each line after the first one is indented with whitespace.

I think you'd want something like this (you need to manually add newlines where you want them)

```
message = (
    "The 'probability_two_random_records_match' has been set to the default value (0.0001)."
    " If this is not the desired behaviour, either: "
    "- assign a value for `probability_two_random_records_match` in your settings dictionary,"
    " or "
    "- estimate with the `linker.estimate_probability_two_random_records_match` function."
)
```

Reply:
Yes, it will do. I mentioned this to Ross privately yesterday.